### PR TITLE
[OWL-1873] lambda query api from query module move to f2e-api

### PR DESCRIPTION
**Pull request by masato25**
1. move code from [query](https://github.com/masato25/lambdaQuery) into f2e-api, because currently query module is messy and not easy maintain anymore.
2. add batch query support of graphv2, but it's only working on staging environment now. because new version graph is not yet update into production.

## 快速測試:
* 需要安裝 docker & docker-compose
cd #{project_root}/docker && docker-compose -f init.yml up -d mysqltest
cd #{project_root}/modules/f2e-api/test/api/lambda_query
sh run_test.sh

### CODE REVIEW

**Comment by hitripod** [comment]
請再補充 Test cases 與 Lambda 的使用說明

**Comment by masato25** [comment]
那這樣要先把測試框架的code先merge進來 :hammer:

**Comment by hitripod** [comment]
Merged: https://github.com/Cepave/open-falcon-backend/pull/367

File: modules/f2e-api/lambda_extends/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# LambdaQuery 可擴充運算聚合模組
+* support open-falcon graph
+
+![](./docs/lambdaQuery.jpg)
```
Comment (hitripod) [inline modules/f2e-api/lambda_extends/ReadMe.md]: 這張圖有什麼含義嗎？

Comment (masato25) [reply modules/f2e-api/lambda_extends/ReadMe.md]: 喔喔,  只是因為從這邊搬過來的
https://github.com/masato25/lambdaQuery

**Pull request merged**